js_modules/dagster-ui/packages/ui-core/src/ticks/SensorDryRunDialog.tsx

```diff
@@ -470,3 +515,27 @@ const Grid = styled.div`
     margin-top: 4px;
   }
 `;
+
+const ComputedCursorGrid = styled.div`
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  padding-bottom: 12px;
+  ${Subheading} {
+    padding-bottom: 4px;
+    display: block;
+  }
+  pre {
+    margin: 0;
+  }
+  button {
+    margin-top: 4px;
+  }
+`;
+
+const SkipReasonNonIdealStateWrapper = styled(Box)`
+  > div:first-child {
+    margin: auto !important;
+    width: unset !important;
+    max-width: unset !important;
```

Comment:
Normally I would only recommend this type of approach (relying on the dom structure of a component) when you don't have control of the component in question (eg. third party libraries like blueprint). In this case since `NonIdealState` is our component we should add a class name and target that here. That will make this coupling easy to find via grep and will make it less likely for someone to unintentionally break this.

Reply:
good find, thanks 🙏